examples/0-counter-part.elm

```diff
@@ -47,7 +45,7 @@ update msg model =
       init
 
     CounterMsg msg' -> 
-      Parts.update CounterMsg msg' model
+      Counters.pass CounterMsg msg' model
```

Comment:
Ah, I see the problem (and difference to the existing solution) now.

I already have an idea of how to fix it, but it involves even more functions. I'll think about it.